openage/convert/driver.py

```diff
@@ -222,6 +223,39 @@ def convert_metadata(args):
             player_palette.save_visualization(outfile)
 
 
+def extract_mediafiles_names_map(files_to_convert, args):
+    """ Gets names from the *.bin files, make them lowercase """
+    if GameVersion.age2_fe in args.game_versions:
+        suffix = '.bina'
+    else:
+        suffix = '.bin'
+
+    names_map = dict()
+
+    for filepath in files_to_convert:
+        filename = b'/'.join(filepath.parts).decode()
+        if filename.endswith(suffix):
+            try:
+                for line in filepath.open():
+                    match = re.match(r"\w+_files\s+(\w+)\s+\w+\s+(\w+)", line)
+                    if match:
+                        groups = match.group(2, 1)
+                        names_map[groups[0]] = groups[1].lower()
+            except UnicodeDecodeError:
+                continue
```

Comment:
why ignore the errors? i think we can avoid them alltogether if we `open(..)` them in `ISO-8859-1` or whatever microsoft used for the encoding.